ruby-challenges.rb

```diff
@@ -31,3 +47,18 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# cheese = 'Pop'
+# just verifying if the problem is the capatilization.
+
+def palindrome(string)
+  if string == string.reverse.capitalize
```

Comment:
Though this works in the case of the given values, please consider a method that would makes all the letters wither lowercase or uppercase to ensure that you would not get an unexpected output for a word like `RACEcar`.